src/careamics/config/data_model.py

```diff
@@ -235,11 +239,41 @@ def std_only_with_mean(self: Self) -> Self:
             If std is not None and mean is None.
         """
         # check that mean and std are either both None, or both specified
-        if (self.mean is None) != (self.std is None):
+        if (self.image_mean and not self.image_std) or (
```

Comment:
Looks like a merge artifact

Reply:
Nope

https://github.com/CAREamics/careamics/blob/15ccfb8ca9b4dfb311c169520a3059d10f863c12/src/careamics/config/data_model.py#L240


But it could be automated `black` tool running. Or ruff?